src/components/widgets/search-widget/search-widget.js

```diff
@@ -0,0 +1,62 @@
+// Docs for Search ui widget
+//https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Search.html
+import { loadModules } from '@esri/react-arcgis';
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import SearchWidgetComponent from './search-widget-components';
+
+const SearchWidget = ({ view }) => {
+  const [searchWidget, setSearchWidget ] = useState();
+
+  const keyEscapeEventListener = (evt) => { 
+    evt = evt || window.event;
+    if (evt.keyCode == 27 && view && searchWidget) {
+      handleCloseSearch();
+    }
+  };
+
+  const handleOpenSearch = () => {
+    loadModules(["esri/widgets/Search"]).then(([Search]) => {
+      const sWidget = new Search({
+        view: view,
+        locationEnabled: false, // don't show the Use current location box when clicking in the input field
+        popupEnabled: false, // hide location popup
+        resultGraphicEnabled: false // hide location pin
+      });
+      setSearchWidget(sWidget);
+
+    }).catch((err) => console.error(err));
+  };
+
+  const handleCloseSearch = () => {
+    view.ui.remove(searchWidget);
```

Comment:
Maybe @faustoperez or @fanny can add something here but I think that they said that it would be great that the globe behind the full screen search started to move before the search disappears, maybe some timeout would do the trick?

Reply:
as @fanny said on slack: `we don't remember thinking about that globe movement before the search dissapears`

Reply:
👍 sorry :/